composer.json

```diff
@@ -13,7 +13,7 @@
     "require": {
         "php": ">=5.6",
         "apishka/cs": "~1.0.0",
-        "composer-plugin-api": "~1.0.0",
+        "composer-plugin-api": "^1.0.0",
```

Comment:
thus, there is nothing making the plugin incompatible with the 1.0 plugin API (especially given it is the current one). So there is no reason to bump the min version